fili-core/src/main/java/com/yahoo/bard/webservice/web/ResponseData.java

```diff
@@ -543,7 +370,7 @@ private static String getDimensionColumnName(Dimension dimension, DimensionField
      *
      * @throws IOException if the generator throws an IOException.
      */
-    private void writeMetaObject(
+    public void writeMetaObject(
```

Comment:
`writeMetaObject` should be pushed onto the two `JSON` request writers, since this is a JSON-specific thing. We may want to combine the JSON and JSONAPI writers into a single hierarchy with a shared parent class, which can have this method.